src/modules/PublicLab.RichTextModule.Table.js

```diff
@@ -61,13 +61,15 @@ module.exports = function initTables(_module, wysiwyg) {
     $("#tableRows").text( Number($("#tableRows").text()) + 1 );
   });
   $(document).on('click', '#decRows', function() {
-    $("#tableRows").text( Number($("#tableRows").text()) - 1 );
+    const numOfRows = Number($("#tableRows").text());
+    if (numOfRows > 1) $("#tableRows").text( numOfRows - 1 );
   });
   $(document).on('click', '#incCols', function() {
     $("#tableCols").text( Number($("#tableCols").text()) + 1 );
   });
   $(document).on('click', '#decCols', function() {
-    $("#tableCols").text( Number($("#tableCols").text()) - 1 );
+    const numOfCols = Number($("#tableCols").text());
+    if (numOfCols > 1) $("#tableCols").text( numOfCols - 1 );
```

Comment:
Hi @NARUDESIGNS, these changes are from another PR. Maybe you forgot to do a `git stash` before switching to another branch.

Reply:
> Hi @NARUDESIGNS, these changes are from another PR. Maybe you forgot to do a `git stash` before switching to another branch.

Hi, sorry I haven't used git stash before but I will look it up and make changes. But before I go on to do that, I need you to help me clarify. Do you mean that I have made these changes in a different PR already?

Reply:
@NARUDESIGNS I mean these changes were made for PR #757.

Reply:
You can also open this branch up, and just overwrite the file with a saved copy of the unmodified file, so that this PR will show only those changes which address it's purpose. Thank you!!! That will help keep each fix in its own PR and separate commits. Thanks!

Reply:
Oh I get it now. I'm sorry for the messy details. I will fix it today and push again.
Thank you and happy new month 🥳

Reply:
Once again, I sincerely apologise for the back on forth on a minimal change like this. I have removed changes that were meant for PR #757 as @TildaDares mentioned and the current commit now carries only the change required for this PR.
Please let me know if this works or if you need me to do a `git reset --hard` instead.
cc @jywarren

Reply:
No worries! Part of the learning process! Let me see now that I merged #757